utils/systemd/monerod.service

```diff
@@ -1,37 +1,87 @@
+### Sample unit file for running monerod as a systemd service ###
+
+### For more information about the options specified in the [Unit] and [Install] sections,
+### please see the systemd.unit(5) man page.
+
 [Unit]
-Description=Monero Full Node
+
+Description=Monero Daemon
 After=network.target
 
+### If monerod is configured to use Tor via "--tx-proxy" and, optionally, "--anonymous-inbound",
+### and if Tor is installed and running as a systemd service,
+### then replace the above "After=network.target" directive with the following two directives:
+
+# After=network.target tor.service
+# Wants=tor.service
+
+### The following [Install] section allows monero.service to be enabled via "systemctl enable",
+### which in turn causes monero.service to be automatically started at system startup.
+### The "Alias=monerod.service" directive allows monero.service to be invoked as monerod.service in various systemctl commands
+### if monero.service is enabled via "systemctl enable".
+
+[Install]
+
+Alias=monerod.service
```

Comment:
This is not needed as already the service name.

Reply:
Oh, right. I forgot about that.